### PR TITLE
docs(css): Add more details about scrollbar-color and scrollbar-width props, hints for fallback

### LINKED ISSUE #32277: Add note regarding `::-webkit-scrollbar` and CSS scrollbar property interaction
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

As of Chromium 121 the standard CSS scrollbar properties are supported, and when both these and the `::-webkit-scrollbar` properties are both supported the standard properties win. While this is common practice for replacing prefixed CSS with standard it has caught some people by surprise.

Because of this it could be useful to add a note explaining this to the MDN page?

### What did you expect to see?

Potentially a small note on the page that explains that when scrollbar-color OR scrollbar-width have a value other than auto, they will override the `::-webkit-scrollbar-*" pseudo styling.

### Do you have any supporting links, references, or citations?

I implemented the standard properties within Chromium so I know this interaction is intended behaviour.

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by bsmth**
### Description

This PR adds some more details to the note about `scrollbar-color` and `scrollbar-width` props including a fallback example.

### Motivation

Some people are not expecting to see styles overridden if `scrollbar-color` and `scrollbar-width` are specified in combination with non-standard `::-webkit-scrollbar` pseudo-elements. 

### Additional details

* https://developer.chrome.com/blog/new-in-chrome-121
* https://developer.chrome.com/docs/css-ui/scrollbar-styling (includes example using `@supports` for fallback)
* https://groups.google.com/a/chromium.org/g/blink-dev/c/PkEsMirl2zE

### Related issues and pull requests

Fixes #32277

* https://github.com/mdn/browser-compat-data/issues/22195

### CODE REVIEW

File: files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md [hunk 1]
```diff
@@ -18,8 +18,12 @@ browser-compat:
 
 The `::-webkit-scrollbar` CSS pseudo-element affects the style of an element's scrollbar when it has scrollable overflow.
 
-> **Note:** The `::-webkit-scrollbar` vendor-prefixed pseudo-element is not supported on all browsers (see [Browser compatibility](#browser_compatibility)).
-> The {{cssxref("scrollbar-color")}} and {{cssxref("scrollbar-width")}} standard properties may be used as an alternative for browsers that do not support this pseudo-element. When these properties are set, `::-webkit-scrollbar` styling is disabled.
+> **Note:**
+> The `::-webkit-scrollbar` vendor-prefixed pseudo-elements are not supported on all browsers (see [Browser compatibility](#browser_compatibility)).
```
Comment (estelle) [inline files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md:22]: two comments, but ok if you don't listen to either:
1) i prefer the singular " vendor-prefixed pseudo-element is not supported" as we're talking about the single pseudo-element here, not the list in the next section. But maybe you're indicating that there will be more (sub) elements below
2) I would consider making the note just regular text.

File: files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md [hunk 2]
```diff
@@ -18,8 +18,12 @@ browser-compat:
...
+> The {{cssxref("scrollbar-color")}} and {{cssxref("scrollbar-width")}} standard properties may be used as an alternative for browsers that do not support these pseudo-elements.
+>
+> If {{cssxref("scrollbar-color")}} and {{cssxref("scrollbar-width")}} are supported and have any value other than `auto` set, they will override `::-webkit-scrollbar-*` styling.
```
Comment (estelle) [inline files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md:25]:
```suggestion

> **Note:** If {{cssxref("scrollbar-color")}} and {{cssxref("scrollbar-width")}} are supported and have any value other than `auto` set, they will override `::-webkit-scrollbar-*` styling.
```
separate into a second note (or make one of the two regular content). These are two different topics

**Review by estelle — COMMENTED** [review]
likely more feedback than you expected. :)

File: files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md:22]: >But maybe you're indicating that there will be more (sub) elements below

Yes, that was the intention, but I'm okay with the singular. I will change it back

**Comment by bsmth** [comment]
Thanks @estelle - incorporated changes based on your comments if you'd like to have another look 👀

File: files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md [hunk 3]
```diff
@@ -47,6 +56,8 @@ You can use the following pseudo-elements to customize various parts of the scro
   width: 10em;
   overflow: auto;
   height: 2em;
+  padding: 1em;
+  margin: auto;
```
Comment (estelle) [inline files/en-us/web/css/_doublecolon_-webkit-scrollbar/index.md:60]:
```suggestion
  margin: 1em auto;
  outline: 2px dashed #ccc;
```
adding a border helps the user identify where to scroll. adding margin top and bottom helps a bit.

**Review by estelle — COMMENTED** [review]
Making two suggestions, but approving not to hold this up:

1) add an outline and margin around the example to help users identify where they should be scrolling.
2) I also recommend adding the words "visible", "invisible" and "custom" at the beginning of each paragraph in the example to help the user identify which one they're looking at.

**Review by estelle — APPROVED** [review]

**Pull request merged**